Eat_and_Greet/src/comps/ChatCard/ChatCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+
+import localstyle from './styles';
+
+interface ChatCardProps {
+  senderName: string;
+  message: string;
+  timestamp: string;
+  onPress?: () => void;
+}
+
+const ChatCard = ({ senderName, message, timestamp, onPress} : ChatCardProps) => {
```

Comment:
I think we could have a better prop name than 'senderName' since it could be a groupchat